app/assets/javascripts/settings-controller.coffee

```diff
@@ -60,35 +62,57 @@ template = """
       <option value={{code}}>{{description}}</option>
       {{/}}
     </select>
-  </div>                           
+  </div>
 </div>
 
 
 <h1 class="settings-subheader">Works in Progress</h1>
-<a><center> Here you can access all the models that have been edited</center></a>
-  
+<div class="description">Here are all the models you have made changes to in NetLogo Web.</div>
+
 <ul>
   {{#each workInProgressLinks}}
-    <li> <a href="{{url}}">{{modelTitle}}</a> <br>Date Last Modified: {{dataAccessed}}</li>
-    <br>
+    <li><a href="{{url}}">{{modelTitle}}</a> {{storageTag}} <div>{{dateAccessed}} at {{timeAccessed}}</div></li>
   {{/each}}
 </ul>
 
 """
+#Formatting Links for HTML display
 formatLinks = (wipStorage) ->
   result = []
 
   for url, model of wipStorage.inProgress
-    if model.title and model.timeStamp
-      formattedUrl = "/launch#http://" + url.replace(/^url:\/\//, '')
+    if model.title and model.timeStamp and not(url.startsWith('disk') or url.startsWith('new')) and model.version == WIP_INFO_FORMAT_VERSION
+      storageTagOutput = ""
+      storageTag = url.split(':').shift();
+      if storageTag !="url"
+        storageTagOutput = " (tag: " + storageTag + ")"
+        formattedUrl = "/launch?storageTag=#{storageTag}##{window.location.protocol}//" + url.replace(/.*:\/\//, '')
+      else
+        formattedUrl = "/launch##{window.location.protocol}//" + url.replace(/^url:\/\//, '')
+
       result.unshift({
         modelTitle: model.title,
         url: formattedUrl,
-        dataAccessed: new Date(model.timeStamp).toLocaleString()
+        dateAccessed: new Date(model.timeStamp).toLocaleDateString('locale', {
+          year: 'numeric',
+          month: 'long',
+          day: 'numeric',
+        }),
+        timeAccessed: new Date(model.timeStamp).toLocaleTimeString('locale', {
```

Comment:
I tried `(new Date()).toLocaleString(undefined, { year: 'numeric', month: 'long', day: 'numeric', hour: '2-digit', minute: '2-digit', second: '2-digit' })` in the browser console and this gave me `'September 6, 2023 at 02:14:04 PM'` which seems like what we want.  Any reason not to combine these two separate bits into one?